### ISSUE TITLE
EmbeddingOptions `printEnabled` doesn't work

### ISSUE THREAD

**Issue by hirouniverse** [awslabs/amazon-quicksight-embedding-sdk#63]
You can pass `printEnabled` option and control whether you display printer button on your embeded dashboard.

In this sdk's document says that;

> ### PrintEnabled element (optional)
> 
> The printEnabled element can be used to enable or disable print option for dashboard embedding. The default value is false. And, if both undoRedo and reset options are disabled, the navbar and print option wont be shown anyways, even if printEnabled is true.

But it seems that Amazon QuickSight deals `printEnabled=true`  when you don't pass `printEnabled` option so that you cannot hide print button on your embeded dashboard because of the inconsistency of specification.

the following code controls the option:

https://github.com/awslabs/amazon-quicksight-embedding-sdk/blob/b02326f1242ccec7e3ea2c1bf7149f1e1da16fa7/src/EmbeddableObject.js#L243-L245

when `printEnabled` is `false`:

URL parameter `pringEnabled` is not set and QuickSight deals the `printEnabled` is `true` and print button is dispalyed (unexpected behavior from the point of view of sdk)

when `printEnabled` is `true`:

URL parameter `printEnabled` is set as `true` and QuickSight receives it and print button is displayed (expected behavior)

We should modify this like:

```js
    if (!printEnabled) {
        src = src + '&printEnabled=' + String(printEnabled);
    }
```

so that SDK can display dashboard correctly.

or maybe it's better to change the QuickSight's specification how to deal `printEnabled` option. (this is out of topic from this sdk)

**Comment by hirouniverse** [comment]
I couldn't find the available query parameter of QuickSight, but I tired with my project and I found this issue.

I could fine only the following link as a reference of available option for QuickSight url parameters.

https://docs.aws.amazon.com/quicksight/latest/APIReference/API_GetDashboardEmbedUrl.html#API_GetDashboardEmbedUrl_ResponseSyntax

**Comment by hirouniverse** [comment]
btw, you can add `printEnabled=false` at the end of URL manually and hide print button if you need.
This is temporary solution for now I'm using.

**Comment by joyarunraj** [comment]
Thank you for bringing this issue to our attention. We will try to address this bug, in the next JS SDK revision.

**Comment by hirouniverse** [comment]
@joyarunraj 
Thank you for your comment.
I made PR for this so please kindly review and use this for release if it is acceptable.
https://github.com/awslabs/amazon-quicksight-embedding-sdk/pull/64

**Comment by joyarunraj** [comment]
This bug has now been addressed with the latest JS SDK release v1.0.13
